Lib/WebmasterTools/WebmasterToolsAuthContainer.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Ik snap het nut van deze klasse niet. 

Hij doet helemaal niets, wat is er dan anders aan dan z'n parent?